pkgs/tools/misc/tlp/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv, fetchgit, makeWrapper, perl, systemd, iw, rfkill, hdparm, ethtool, inetutils, kmod
+, enableRDW ? true, networkmanager }:
+
+stdenv.mkDerivation {
+  inherit enableRDW;
+
+  name = "tlp-0.6";
+
+  src = fetchgit {
+    url = "https://github.com/linrunner/TLP";
+    rev = "dbcc09e0f1730e7b85c7387dcc06d01f1547edba";
```

Comment:
Yeah, shouldn't be a problem I hope using `fetchFromGitHub` and using a release such as this one: https://github.com/linrunner/TLP/releases/tag/0.6